File: python/repeticao_sequencial_de_numero_3.py
"""
Crie um aplicativo que mostre na tela a seguinte contagem:
0 3 6 9 12 15 18 Acabou! 
"""


def repetir_numero():
    valor = int(input('Digite o valor 0 por favor '))
    lista = []
    for valor in range(valor, 18+1, 3):
        lista.append(valor)
    textoLIsta = " ".join(map(str, lista))
    print(f'{textoLIsta} Acabou')


repetir_numero()

File: js/repeticao_sequencial_de_numero_3.js
/**
 * Crie um aplicativo que mostre na tela a seguinte contagem:
    0 3 6 9 12 15 18 Acabou! 
 */

let repetirNumero = () => {
    let valor = 0
    let lista = []
    console.log(valor)
    for(let i = 0; i <= 18; i=i+3) {
        lista.push(i)
    }
    let lista_texto = lista.toString()
    console.log(`${lista_texto} acabou`)
}
repetirNumero()

//problemas em colocar um valor variável no for...